### PR TITLE
Tweaks for clarity, English, structure

### PR DESCRIPTION
**Pull request by bpbond**

### CODE REVIEW

File: CriticalGF.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ Global forests provide critical habitat for many species, essential ecosystem se
 
 ## Introduction
 
-Forests are one of the most important ecosystems on earth, providing habitat for a large proportion of species and contributing extensively to global biodiversity [@Crowther2015a]. In the previous century human activities have influenced global bio-geochemical cycles [@Canfield2010; @Bonan2008], with one of the most dramatic changes being the replacement of 40% of Earth’s formerly biodiverse land areas with landscapes that contain only a few species of crop plants, domestic animals and humans [@Foley2011]. These local changes have accumulated over time and now constitute a global forcing [@Barnosky2012]. [TODO: not sure I agree with this sentence..."grown" over time?]
```
Comment (lsaravia) [inline CriticalGF.md]: @bpbond Please check this accumulated vs grown

File: CriticalGF.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ The largest patch is the one that connects the highest number of sites in the ar
...
 
-The $RS_{max}$ is a useful qualitative index that does not [TODO: does not what? work?] if the system is near or far from the critical transition; this can be evaluated using the temporal fluctuations. We calculate the fluctuations around the mean with the absolute values $\Delta S_{max}=S_{max}(t)-\langle S_{max} \rangle$, using the proportions of $RS_{max}$. To characterize fluctuations we fitted three empirical distributions: power-law, log-normal, and exponential, using the same methods described previously. We expect that large fluctuation near a critical point have heavy tails (log-normal or power-law) and that fluctuations far from a critical point have exponential tails, corresponding to Gaussian processes [@VanRooij2013]. As the data set spans 15 years, we do not have enough power to reliably detect which distribution is better [@Clauset2009]. To improve this we performed the goodness of fit test described above for all the distributions. We generated animated maps showing the fluctuations of the two largest patches to aid in the interpretations of the results.
```
Comment (lsaravia) [inline CriticalGF.md]: I added  "tell us" may be another expression is better

**Comment by bpbond** [comment]
Looks good to me.

**Comment by lsaravia** [comment]
Thus I prepare the submission to Global Change Biology!

**Pull request merged**